pkg/action/cl001/ac000/plan.go

```diff
@@ -13,22 +11,22 @@ import (
 var Plan = []plan.Step{
 	{
 		Action:  "ac001",
-		Backoff: backoff.NewConstant(24*time.Minute, 3*time.Minute),
+		Backoff: plan.NewBackoff(24*time.Minute, 3*time.Minute),
```

Comment:
I did not want to "pollute" the public backoff interface for this. So in order to have the time information available we need this wrapper. I do not expect that we need more fancy variants of the backoff implementation.